virttest/env_process.py

```diff
@@ -804,11 +804,22 @@ def preprocess(test, params, env):
                 params[name_tag] = os.path.join(image_nfs.mount_dir,
                                                 image_name_only)
 
-    # firewall blocks dhcp from guest through virbr0
-    if params.get('firewalld_dhcp_workaround', "no") == "yes":
-        firewall_cmd = utils_iptables.Firewall_cmd()
-        if not firewall_cmd.add_service('dhcp', permanent=True):
-            logging.error('Failed to add dhcp service to be permitted')
+    firewalld_service = params.get('firewalld_service')
```

Comment:
we can instead have it here once`firewalld = utils_iptables.Firewalld()`

Reply:
Yep, I thought about it but don't think we should initialize it unless we know it's gona be used.